extensions/google-drive/src/constants.ts

```diff
@@ -28,3 +29,9 @@ export const MAX_TMP_FILE_PREVIEWS_LIMIT = 500; // Average size of a file previe
 export const DB_FILE_PATH = join(homedir(), ".raycast-google-drive-sqlite.db");
 export const TMP_FILE_PREVIEWS_PATH = join(tmpdir(), "raycast-google-drive-file-previews");
 export const FILES_LAST_INDEXED_AT_KEY = "filesLastIndexedAt";
+export const FD_PATH = path.join(environment.supportPath, "fd");
+export const FD_ARCHIVE_URL =
+  "https://github.com/sharkdp/fd/releases/download/v8.3.2/fd-v8.3.2-x86_64-apple-darwin.tar.gz";
```

Comment:
Does this binary run on M1 natively or will it have to use Rosetta?

Reply:
My mac runs on a Intel chip but I'm hopeful that it can work on an M1 as well.